sql/moz-fx-data-shared-prod/telemetry_derived/desktop_engagement_clients_v1/query.sql

```diff
@@ -12,20 +12,22 @@ SELECT
   cfs.attribution_dlsource,
   cfs.attribution_medium,
   cfs.attribution_ua,
+  cfs.attribution_experiment,
```

Comment:
we need both `attribution_experiment` and `attribution_variation`. Which I know we don't have on first_seen but we should have on last_seen.

Reply:
done